conf/backup_method

```diff
@@ -17,59 +18,52 @@ do_need_mount() {
     true
 }
 
-LOGFILE=/var/log/backup_borg.err
-log_with_timestamp() {
-    sed -e "s/^/[$(date +"%Y-%m-%d_%H:%M:%S")] /" | tee -a $LOGFILE
-}
-
 do_backup() {
     export BORG_PASSPHRASE
+    export BORG_REPO
     export BORG_RSH
+    export BORG_LOGGING_CONF
     export BORG_RELOCATED_REPO_ACCESS_IS_OK=yes
     work_dir="$1"
     name="$2"
-    repo="$3"
-    size="$4"
-    description="$5"
-    current_date=$(date +"%Y-%m-%d_%H:%M")
-    pushd "$work_dir"
+    size="$3"
+    description="$4"
     set +e
-    if "$borg" init -e repokey "$repo" ; then
+    if ! "$borg" config -l > /dev/null 2>&1; then
+        "$borg" init -e repokey
         # human_size=`echo $size | awk '{ suffix=" KMGT"; for(i=1; $1>1024 && i < length(suffix); i++) $1/=1024; print int($1) substr(suffix, i, 1), $3; }'`
         # Speed in Kbps
         # speed=1000
         # evaluated_time=$(($size / ($speed * 1000 / 8) / 3600))
         echo "Hello,
 
-Your first backup on $repo is starting.
+Your first backup on $BORG_REPO is starting.
 
 This is an automated message from your beloved YunoHost server." | /usr/bin/mail.mailutils -a "Content-Type: text/plain; charset=UTF-8" -s "[YNH] First backup is starting" "root"
     fi
     set -e
 
-    "$borg" create "$repo::_${name}-${current_date}" ./ 2>&1 >/dev/null | log_with_timestamp
-    popd
-
-    # About thi _20 it's a crazy fix to avoid pruning wordpress__2
-    # if you prune wordpress
-    "$borg" prune "$repo" -P "_${name}-" --keep-hourly 2 --keep-daily=7 --keep-weekly=8 --keep-monthly=12 2>&1 >/dev/null | log_with_timestamp
+    # About the {now} placeholder:
+    # https://borgbackup.readthedocs.io/en/stable/usage/create.html#description
+    # In the archive name, you may use the following placeholders: {now}, {utcnow}, {fqdn}, {hostname}, {user} and some others.
+    "$borg" create --stats "::${name}-{now}" "$work_dir"
 
-    # Prune legacy archive name without error on wordpress/wordpress__2
-    "$borg" prune "$repo" -P "${name}_" --keep-within 2m --keep-monthly=12 2>&1 >/dev/null | log_with_timestamp
+    "$borg" prune --glob-archives "${name}-*" --list --keep-hourly 2 --keep-daily=7 --keep-weekly=8 --keep-monthly=12
 
     # We prune potential manual backup older than 1 year
-    "$borg" prune "$repo" --keep-within 1y 2>&1 >/dev/null | log_with_timestamp
+    "$borg" prune --list --keep-within 1y
```

Comment:
@kay0u I'm not very fond of pruning without the user being informed ?
Shouldn't this "1y" be a setting in the config panel, set to "never" by default ?

Reply:
This code was already there, it's a discussion for another PR. To me it's very reasonable to have pruning by default because you don't want backup to just pile up and eat space forever. 1 year is a reasonable value ... Most of the time when you need to restore a backup, you need one from say a few days ago ...

Reply:
I understand your point. It should at least be mentionned somewhere in the post-install notes for instance.
I'll try to see if I can work on this small matter.